pkg/services/auth/id.go

```diff
@@ -19,3 +21,9 @@ type IDSigner interface {
 type IDClaims struct {
 	jwt.Claims
 }
+
+const settingKey = "grafanaIdForward"
```

Comment:
nit: I would name this `forwardIDToken`

Reply:
sorry, I meant the actual setting key, the one that data sources need to set

Reply:
Yeah that makes more sense :)

Reply:
maybe better to be `forwardGrafanaIdToken` (at the beginning I thought this was forwarding `X-ID-Token`)